calkit/cli/new.py

```diff
@@ -1150,21 +1153,34 @@ def new_conda_env(
         if project_name is None:
             project_name = os.path.basename(os.getcwd())
         conda_name = calkit.to_kebab_case(project_name) + "-" + name
-    # Write environment to path
-    _check_path_dir(path)
-    conda_env = dict(
-        name=conda_name, channels=["conda-forge"], dependencies=packages
-    )
-    if prefix is not None:
-        from calkit.cli.main import ignore
+    if packages is not None:
+        assert isinstance(packages, list)
+        # Write environment to path
+        _check_path_dir(path)
+        conda_env = dict(
+            name=conda_name, channels=["conda-forge"], dependencies=packages
+        )
+        if prefix is not None:
+            from calkit.cli.main import ignore
 
-        conda_env["prefix"] = prefix
-        ignore(prefix, no_commit=True)
-        repo.git.add(".gitignore")
-    if pip_packages:
-        conda_env["dependencies"].append(dict(pip=pip_packages))
-    with open(path, "w") as f:
-        ryaml.dump(conda_env, f)
+            conda_env["prefix"] = prefix
+            ignore(prefix, no_commit=True)
+            repo.git.add(".gitignore")
+        if pip_packages:
+            conda_env["dependencies"].append(dict(pip=pip_packages))
+        with open(path, "w") as f:
+            ryaml.dump(conda_env, f)
+    elif packages is None and os.path.isfile(path):
+        with open(path) as f:
+            conda_env: dict = ryaml.load(f)
+        # Remove prefix
+        conda_env.pop("prefix", None)
```

Comment:
@rebeccamccabe curious if this would be a problem for your use case. I'm trying to make the environment more portable by removing any prefix, in case someone exported one with `conda env export`.